tests/validation/tests/v3_api/common.py

```diff
@@ -664,9 +664,16 @@ def get_schedulable_nodes(cluster, client=None, os_type=TEST_OS):
     schedulable_nodes = []
     for node in nodes:
         if not node.unschedulable:
-            for tkey, tval in node.taints.items():
-                if tval == "NoExecute" or tval == "NoSchedule":
-                    break
+            shouldSchedule = True
+            try:
+                for tval in node.taints:
+                    if str(tval).find("NoExecute") > -1 or str(tval).find("NoSchedule") > -1:
```

Comment:
Do we want to disallow scheduling if the value is `PreferNoSchedule` then too?

Reply:
No, good catch. Updated in this latest version.